requirements.txt

```diff
@@ -1,6 +1,6 @@
 six
 numpy>= 1.11
-pandas>=0.13.1
+pandas==0.19.2
```

Comment:
Pinning a pandas version will be inconvenient for regular users. If we need this for kubeface training we should do it in a way that doesn't impact the main mhcflurry requirements for everyone